docs/tutorial.rst

```diff
@@ -59,24 +61,28 @@ Your shell prompt should change once you are in the virtual env to show (cci) at
 
 If you would like to automatically activate the virtual environment whenever you open a new terminal, you can add the activate command to your ``~/.bash_profile``.
 
-For more information about virtual environments in Python, see the Python Packaging User Guide: https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments
-
 Windows
 ^^^^^^^
 
+Ensure you are in your home directory (C:\Users\your.username) and execute the following commands:
+
 .. code-block:: powershell
 
-    python -m venv ~\cci
-    ~\cci\Scripts\Activate.ps1
+    python -m venv cci
+    cci\Scripts\activate
 
 .. note::
    If this fails because you don't have the ability to run scripts,
    try running: ``Set-ExecutionPolicy Unrestricted -Scope CurrentUser``
 
+Similar to the macOS commands above, this creates a virtualenv in a new folder named ``cci`` and activates it.
+
+If you would like to automatically activate the virtual environment whenever you open a new terminal, you can add the activate command to your ``~/.bash_profile``.
```

Comment:
This paragraph doesn't apply to Windows unless you're using bash somehow, which is not what we're assuming above.

Reply:
I'll re-run through the tutorial and confirm these instructions again. I use Git Bash by default, since we aren't allowed to use PowerShell.

Reply:
@robertbwatson Jason wrote up docs for setting up CumulusCI with VSCode and Git Bash for the community sprint, and I think the intent is to turn them into updated installation documentation here. Those have gotten a lot more real world testing.

Reply:
@davisagli Yep, I've been using those and helped Jason for the initial walk-through.

I just made another commit with some updates here, but if a larger documentation update is coming soon that's great too.